sky/clouds/service_catalog/data_fetchers/fetch_lambda_cloud.py

```diff
@@ -0,0 +1,119 @@
+"""A script that generates the Lambda Cloud catalog.
+
+Usage:
+    python fetch_lambda_cloud.py [-h] [--api-key API_KEY]
+                                 [--api-key-path API_KEY_PATH]
+
+If neither --api-key nor --api-key-path are provided, this script will parse
+`~/.lambda/lambda_keys` to look for Lambda api key.
+"""
+import argparse
+import csv
+import json
+import os
+import requests
+
+ENDPOINT = 'https://cloud.lambdalabs.com/api/v1/instance-types'
+LAMBDA_KEYS_PATH = os.path.expanduser('~/.lambda_cloud/lambda_keys')
+
+# This is the list that Lambda Labs gave us.
+regions = [
```

Comment:
nit: Why don't we use caps as it is a constant?

Reply:
Good idea -- fixed.